traffic_ops/traffic_ops_golang/cdn_lock/cdn_lock.go

```diff
@@ -1,3 +1,4 @@
+// cdn_lock package contains the CRD methods related to CDNLocks
```

Comment:
Package GoDocs should be complete sentences that start with `Package {{package name}}`.